config/brokerSets.json

```diff
@@ -0,0 +1,14 @@
+{
+  "brokerSets":[
```

Comment:
I'm assuming this file is only for testing purpose. Do we want to commit this pre-defined brokerSets file?

Reply:
The test file is under test resources. This is basically to provide a default implementation if someone wants to use a file based implementation of BrokerSets. Just like you would find in CapacityResolver.

Reply:
I feel like leave a placeholder here is good enough. E.g. something like

"
{
      "brokerSetId": "0",
      "brokerIds": [0],
      "doc": "This block contains broker ids that belong to the BrokerSet 0"
    },
"


@efeg  what do you think?

Reply:
updated